R/ts_generator.R

```diff
@@ -0,0 +1,39 @@
+#' Piecewise linear decay
+#'
+#' @param t Times to simulate
+#' @param offset Offset
+#' @param pert Perturbation intensity (signed)
+#' @param tpert Perturbation timing
+#' @param recover Perturbation half-life
+#'
+#' @return The time series
+#' @export
+#'
+#' @examples
+piecewise <- function(t, offset = 0, pert = 0, tpert = 0, recover = 1) {
```

Comment:
the piecewise function is not tested

Reply:
1f8dc90 fixes this.

The tests are still quite basic, but good enough for merging.